.github/workflows/libssl.yaml

```diff
@@ -209,9 +212,6 @@ jobs:
           - container: ubuntu
             version: 24.04
             package: deb
-        # nb. ubuntu:22.04 has a too-old golang to build nfpm
-        # we could install an upstream golang if we want to
-        # package there.
```

Comment:
I removed this comment because it doesn't apply anymore now that we're installing nfpm from releases and not src.

I didn't add a 22.04 container because it felt like the value is pretty minimal. WDYT?